test/functional/activities_controller_test.rb

```diff
@@ -105,7 +107,7 @@ def test_previous_project_index
                }
   end
 
-  def test_user_index
+  it 'user_index' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.